rundeckapp/grails-spa/packages/ui-trellis/src/library/components/plugins/tests/pluginPropView.spec.ts

```diff
@@ -0,0 +1,138 @@
+    const selectedValues = wrapper.findAll(".text-success");
+    expect(selectedValues.length).toBe(2);
+    expect(selectedValues[0].text()).toContain("Option1");
+    expect(selectedValues[1].text()).toContain("Option2");
+  });
+  it("renders multi-line values using the Expandable component", async () => {
+    const wrapper = createWrapper({
+      prop: {
+        type: "String",
+        title: "Multiline Value",
+        desc: "Description for multiline",
+        options: { displayType: "MULTI_LINE" },
+      },
+      value: "Line 1\nLine 2\nLine 3",
+    });
+    const expandable = wrapper.findComponent(Expandable);
+    const label = expandable.find("[title='Description for multiline']");
+    expect(label.text()).toContain("Multiline Value:");
+    const lineCount = expandable.find(".text-info");
+    expect(lineCount.text()).toContain("3 lines");
+  });
+  it("renders ACE Editor for code syntax mode", async () => {
+    const wrapper = createWrapper({
+      prop: {
+        type: "String",
+        title: "Code Snippet",
+        desc: "Code description",
+        options: { displayType: "CODE", codeSyntaxMode: "javascript" },
+      },
+      value: "console.log('Hello World');",
+    });
+    const expandable = wrapper.findComponent(Expandable);
+    const aceEditor = expandable.findComponent(AceEditor);
+    expect(aceEditor.props("lang")).toBe("javascript");
+    expect(aceEditor.props("readOnly")).toBe(true);
+    expect(aceEditor.props("modelValue")).toBe("console.log('Hello World');");
+  });
+  it("renders DYNAMIC_FORM type property with custom attributes", async () => {
+    const wrapper = createWrapper({
+      prop: {
+        type: "String",
+        title: "Dynamic Form",
+        desc: "Dynamic description",
+        options: { displayType: "DYNAMIC_FORM" },
+      },
+      value: JSON.stringify([
+        { label: "Custom Label 1", value: "Custom Value 1" },
+        { label: "Custom Label 2", value: "Custom Value 2" },
+      ]),
+    });
+    const customPairs = wrapper.findAll('[data-testid="configpair"]');
+    expect(customPairs.length).toBe(2);
+    expect(customPairs[0].text()).toContain("Custom Label 1:");
+    expect(customPairs[0].text()).toContain("Custom Value 1");
+    expect(customPairs[1].text()).toContain("Custom Label 2:");
+    expect(customPairs[1].text()).toContain("Custom Value 2");
+  });
+  it("renders password type with obfuscated value", async () => {
+    const wrapper = createWrapper({
+      prop: {
+        type: "String",
+        title: "Password",
+        desc: "Sensitive data",
+        options: { displayType: "PASSWORD" },
+      },
+      value: "secret",
+    });
+    const title = wrapper.find("[title='Sensitive data']");
+    expect(title.exists()).toBe(true);
+    expect(title.text()).toContain("Password:");
+    const obfuscatedText = wrapper.find(".text-success");
+    expect(obfuscatedText.text()).toBe("••••••••••••");
+  });
+});
```

Comment:
suggestion: would be good to add one more test to cover the template case defined from lines 19-37 on pluginPropView that renders an icon.